<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDifficulty</class>
 <widget class="QDialog" name="DialogDifficulty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_row">
       <property name="text">
        <string>行数</string>
       </property>
       <property name="buddy">
        <cstring>spinBoxRow</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxRow">
       <property name="minimum">
        <number>8</number>
       </property>
       <property name="maximum">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_column">
       <property name="text">
        <string>列数</string>
       </property>
       <property name="buddy">
        <cstring>spinBoxColumn</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxColumn">
       <property name="minimum">
        <number>8</number>
       </property>
       <property name="maximum">
        <number>30</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_mine">
       <property name="text">
        <string>雷数</string>
       </property>
       <property name="buddy">
        <cstring>spinBoxMine</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxMine">
       <property name="minimum">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOK">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spinBoxRow</tabstop>
  <tabstop>spinBoxColumn</tabstop>
  <tabstop>spinBoxMine</tabstop>
  <tabstop>pushButtonOK</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonOK</sender>
   <signal>clicked()</signal>
   <receiver>DialogDifficulty</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>DialogDifficulty</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
